index.html

```diff
@@ -0,0 +1,874 @@
+                <h6><a class="nav-link text-nowrap" href="#">U.S. <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">World <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Business <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Tech & Science <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Culture <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Newsgeek <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Sports <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Health <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">The Debate <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Vantage <span class="large text-muted"> | </span><span class="short"> > </span></a></h6>
+              </li>
+              <li class="nav-item">
+                <h6><a class="nav-link text-nowrap" href="#">Weather <span class="short"> > </span></a></h6>
+              </li>
+        </ul>
+        <form class="form-inline my-2 my-lg-0 input-group d-none d-lg-flex">
+          <input class="form-control mr-sm-2 input-group-text" type="search" placeholder="Search" aria-label="Search">
+          <button class="btn my-2 my-sm-0 input-group-append" type="submit"><i class="material-icons">search</i></button>
+        </form>
+      </div>
+    </nav>
+    <!-- Main section  -->
+  <main class="grid-area bg-white">
+      <!-- Top story -->
+      <article class="top-story mb-4 bg-white">
+        <h5 class="text-muted text-uppercase">Top Story</h5>
+        <img class="img-fluid" src="img/bootstrap-top.jpg" alt="Bootstrap">
+        <h2 class="text-weight-bold">Bootstrap makes building a site faster, developers say</h2>
```

Comment:
- [ ] The elements inside your `<main>` tag should have more margin to the sides. Please use bootstrap classes for this.